modules/ROOT/pages/hosting-options.adoc

```diff
@@ -100,7 +100,27 @@ By default, no value is returned for `dlbUrl` if you haven’t previously config
 +
 `datagraph load-balancer-config add <url>`
 +
-Replace `<url>` with a valid URL that includes the DLB name and domain (gathered in Steps 3 and 4 of the gathering details task) and the mapping rule *Input Path*, for example,
+Replace `<url>` with a valid URL that includes the DLB name and DataGraph app name (gathered in Steps 3 and 4 of the gathering details task) and the mapping rule *Input Path*. 
++
+For example, consider the following mapping rules for `examplelb.lb.anypointdns.net` depending on the value of *Input Path*:
```

Comment:
I think it would be crearer if we use a real app name in the examples, like:

For example, consider the following mapping rules for `examplelb.lb.anypointdns.net` depending on the value of *Input Path*, and the Datagraph App Name as `datagraph-xz082edx-1585-901xe374-d0x0`